submissions/DmytryjK/js-oop-exercise-tiny/index.js

```diff
@@ -0,0 +1,96 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+
+// ======== OUTPUT ========
+/* Use print(message) for output.
+   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
+
+   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
+   However, please, REFRAIN from improving visuals at least until your code is reviewed
+   so code reviewers might focus on a single file that is index.js.
+   */
+
+/* Print examples:
+   print('ABC');
+   print('<strong>ABC</strong>');
+   print('<strong>ABC</strong>', 'div');
+
+   print('human; John; male; 2; 2; Hello world!; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
+   */
+
+class Inhabitant {
+   constructor(name, spacies, gender, saying) {
+      this.name = name;
+      this.spacies = spacies;
+      this.gender = gender;
+      this.saying = saying;
+      this.property = ['name', 'spacies', 'gender', 'saying'];
+   }
+   get props() {
+      return this.property.map(key => this[key]).join('; ');
+   }
+}
+
+class Human extends Inhabitant {
+   constructor (name, gender, saying, spacies, legs, hands) {
+      super(name, spacies, gender, saying);
+      this.spacies = 'human';
```

Comment:
(1) We receive `spacies` from the caller. We send it to the super class. We assign it directly. Something is not required.

(2) If we hardcode species here. WHy do we need class user to specify any at all? Why not to send the hardcoded value to the super class constructor, as it belongs to it, and let the super class handle it as they like it?